kotlin-example/src/main/java/com/moengage/segment/kotlin/sampleapp/AnalyticsKotlinActivity.kt

```diff
@@ -21,10 +22,38 @@ class AnalyticsKotlinActivity : AppCompatActivity() {
             append("MoEngage App Id: ")
             append(BuildConfig.MOENAGE_APP_ID)
         }
+        findViewById<Button>(R.id.button_identify_1).setOnClickListener {
+            analytics.identify("abc1@example.com", getTraitsWithoutUniqueId())
+        }
+        findViewById<Button>(R.id.button_identify_2).setOnClickListener {
+            analytics.identify(getTraitsWithoutUniqueId())
+        }
+        findViewById<Button>(R.id.button_identify_3).setOnClickListener {
+            analytics.identify(getTraits())
+        }
+        findViewById<Button>(R.id.button_track).setOnClickListener {
+            analytics.track("ON_CREATE")
+            analytics.track("TRACK_LOCATION", getEventProperties())
+        }
+        findViewById<Button>(R.id.button_alias).setOnClickListener {
+            analytics.alias("abc1@example.com")
```

Comment:
Please take a different id here because if id will be the same for identify & setAlias. We will have to change everytime while testing setAlias method otherwise it will not work

Reply:
No, It is not same in identify it is abc@example.com and in setAlias is abc1@example.com

Reply:
Done